.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,17 +7,17 @@ about: Create a report to help us fixing breaking things
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Include steps to reproduce the behavior.
+Include steps to reproduce the behaviour.
 
 Example:
 
-1. Create a new React App using `create-react-app`
-2. Install the `@dxc-technology\halstack-react` dependencies and run the app
-3. Click on '....'
-4. Scroll down to '....'
-5. See error
+1. Create a new React App using `create-react-app`.
+2. Install the `@dxc-technology\halstack-react` dependencies and run the app.
+3. Click on '...'.
+4. Scroll down to '...'.
+5. See the error.
 
-**Expected behavior**
+**Expected behaviour**
 A clear and concise description of what you expected to happen.
```

Comment:
Maybe it should be better to put: `A clear and concise description of what you expect to happen.`. Changing `expected` to `expect`.

Reply:
I think is well written, since what we were trying to say is that he/she should explain what they expected to happen when they executed the code in the past.